Bookish/Views/Catalog/BookList.cshtml

```diff
@@ -27,13 +41,17 @@
                 <td>@b.ISBN</td>
                 <td>@b.DatePublished</td>
                 <td>@b.CopyList.Count</td>
+                <td><button class="btn-outline-info"><i class="bi bi-pencil"></i></button></td>

```

Comment:
Just like you've used classes here! (following on from my above comment about using Bootstrap)